Automattic/wp-calypso
# Adds SIWA to social logins page

Requires server patch D31670

#### Changes proposed in this Pull Request

* Adds Sign in with Apple to the social logins page

#### Testing instructions

* Have WordPress.com account
* Go to http://calypso.localhost:3000/me/security/social-login
* Next to Apple click `Connect` button
* Follow steps in the Apple screens to connect account
* On `/me/security/social-login`, observe button now says `Disconnect`
* Click Disconnect button
* Follow steps in Apple screens
* On `/me/security/social-login`, observe button now says `Connect`
* Follow instructions in the server patch
